backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/graphicpromotion/internal/BpkGraphicPromoImpl.kt

```diff
@@ -296,7 +373,11 @@ private class InteractiveBackgroundIndicationNode(private val interactionSource:
 }
 
 private const val RATIO_PORTRAIT: Float = 3 / 4f
-private const val SPONSOR_LOGO_HEIGHT = 60
+private const val RATIO_PORTRAIT_SMALL_TABLET: Float = 464 / 617f
+private const val RATIO_PORTRAIT_TABLET: Float = 705 / 360f
+private const val RATIO_PORTRAIT_DESKTOP: Float = 1024 / 460f
+private const val SPONSOR_LOGO_HEIGHT = 32
+private const val SPONSOR_LOGO_WIDTH = 160
```

Comment:
I'm concerned about these magic numbers. Can we reuse any of the Backpack standard sizes instead?

Reply:
I think these are fine - they are specific to the component. We mainly just have spacing tokens, but these are not spacing tokens and just because spacing gets changed we don't want to change the size of the image